README.md

```diff
@@ -153,7 +153,8 @@ module.exports = {
 
 The ideal configuration for Babel will depend on your project. See [Babel's docs](https://babeljs.io/docs/en/) for more details.
 
-<details><summary markdown="span"><strong>Making your Babel config jest-aware</strong></summary>
+<details>
+  <summary markdown="span"><strong>Making your Babel config jest-aware</strong></summary>
```

Comment:
In MDX v2 this does not compile. 
This is the readme though so not a big deal but I updated it too because mdx docs have the exact same content.

Reply:
This change can be moved to a preparation PR and would work with both MDX v1 and v2